src/clj_github/repository.clj

```diff
@@ -195,3 +198,21 @@
     (fetch-body! client {:path   url
                          :method :put
                          :body   params})))
+
+(defn clone
+  "Download github repository and put content on destination path"
+  ([client org repo dest]
+   (clone client org repo "master" dest))
+  ([client org repo tag dest]
+   (let [clone-path (str (fs/temp-dir "clone-repo") "/")
```

Comment:
If we rely on `io/file` two parameter arity we don't need to concatenate `/`. I think it makes the code easier to read.